src/main/java/de/tum/in/www1/artemis/domain/GradeStep.java

```diff
@@ -27,6 +27,8 @@ public class GradeStep extends DomainObject {
 
     private static final Logger log = LoggerFactory.getLogger(GradeStep.class);
 
+    public static final String PLAGIARISM_GRADE = "U";  // "U" stands for "Unterschleif"
```

Comment:
Maybe just assign this variable the value "Unterschleif", which could be easier to read

Reply:
@krusche told me that the name of the grade is "U" (I guess because it serves as a letter grade like "A", "B", etc.) so as far as I understand instructors would expect to see it as "U".